app/src/main/java/de/danoeh/antennapod/fragment/QueueFragment.java

```diff
@@ -469,6 +471,27 @@ private void onFragmentLoaded() {
         // we need to refresh the options menu because it sometimes
         // needs data that may have just been loaded.
         getActivity().supportInvalidateOptionsMenu();
+
+        // refresh status bar
+        if(queue.size() > 0) {
+            int durationSec = 0;
+            for(FeedItem item : queue) {
+                if(item.getMedia() != null) {
+                    durationSec += item.getMedia().getDuration() / 1000;
+                }
+            }
+            String duration = "";
```

Comment:
minor comment:
How about, instead of the if/else block that follows, using this instead: 

```java
hrs = durationSec / 3600
mins = (durationSec % 3600) / 60
if (hrs)
    duration += (hrs + " " + getString(R.string.time_unit_hours) + " ")
duration += (mins + " " + getString(R.string.time_unit_minutes))
```
Seems clearer, but possibly slightly less efficient if durationSec < 3600 (needless division and modulo)